src/ThermalScattering.tex

```diff
@@ -125,7 +125,32 @@ \subsubsection{\textsf{ITXE} Block}\label{sec:ITXEBlock}
 
-\todo[inline]{Describe the format for $\nxs{7}=2$.}
+When $\nxs{7}=2$, the distribution of outgoing energies for each incident energy is continuous in energy and specified by a probability density function and cumulative distribution function. The format of the \Block{ITXE} in this case is given in \Tableref{tab:ITXEBlockContinuousHeader} and \Tableref{tab:ITXEBlockContinuousData}. As before, the index at the start of the block is \startblock{ITXE}=\jxs{3}. Unlike in the $\nxs{7} < 2$ cases, the number of outgoing energies for each incident energy is allowed to vary. The number of discrete cosines, $N_\mu=$\nxs{3}-1, remains the same for each pair of incident and outgoing energies, however.
+
+\begin{XSSTable}{Start of \Block{ITXE} for $\nxs{7}=2$}
+  \startblock{ITXE}          & \var{L}$(l),l=1,\ldots,N_{in}$  & Location in \XSS\ of distribution for incident energy $l$ \\
+  \startblock{ITXE}+$N_{in}$ & $N_E(l),l=1,\ldots,N_{in}$      & Number of outgoing energies for incident energy $l$
+  \label{tab:ITXEBlockContinuousHeader}
+\end{XSSTable}
+
+\begin{XSSTable}{\Block{ITXE} for $\nxs{7}=2$ (continued)}
+  \var{L}(1)+1                     & $E_1^{out}[E_{in}(1)]$                     & First of \var{NIEB}\tnotex{tn:nieb}\ outgoing energies for inelastic scattering at $E_{in}(1)$    \\
+  \var{L}(1)+2                     & $\mathrm{PDF}_1[E_{in}(1)]$          & Probability density function value for $E_1^{out}[E_{in}(1)]$ \\
+  \var{L}(1)+3                     & $\mathrm{CDF}_1[E_{in}(1)]$          & Cumulative distribution function value for $E_1^{out}[E_{in}(1)]$ \\
+  \var{L}(1)+4                     & $\mu_l(1\rightarrow 1), l=1,\ldots,N_\mu$  & Discrete cosines for scattering from $E_{in}(1)$ to $E_1^{out}[E_{in}(1)]$    \\
+  \var{L}(1)+4+$N_\mu$             & $E_2^{out}[E_{in}(1)]$                     & Second of \var{NIEB} outgoing energies for inelastic scattering at $E_{in}(1)$    \\
+  \var{L}(1)+5+$N_\mu$             & $\mathrm{PDF}_2[E_{in}(1)]$          & Probability density function value for $E_2^{out}[E_{in}(1)]$ \\
+  \var{L}(1)+6+$N_\mu$             & $\mathrm{CDF}_2[E_{in}(1)]$          & Cumulative distribution function value for $E_2^{out}[E_{in}(1)]$ \\
+  \var{L}(1)+7+$N_\mu$             & $\mu_l(1\rightarrow 2), l=1,\ldots,N_\mu$  & Discrete cosines for scattering from $E_{in}(1)$ to $E_2^{out}[E_{in}(1)]$    \\
+  \multicolumn{1}{c}{\vdots}       & \multicolumn{1}{c}{\vdots}                 & \multicolumn{1}{c}{\vdots}                                                    \\
+  \var{L}(1)+1+($N'$-1)(3+$N_\mu$) & $E_{N'}^{out}[E_{in}(1)]$                     & Last of \var{NIEB} outgoing energies for inelastic scattering at $E_{in}(1)$    \\
```

Comment:
What is `N'` here and why are we only going to `N'-1`? If I remember my own reverse engineering, the last pdf/cdf value is not given, but implied. But we would still need mu-values for them, so perhaps I'm not remembering correctly. Am I missing something?

Reply:
Ah yes, that shouldn't be `N'`. It should be the number of outgoing energies for the first incoming energy, i.e. `N_E(1)`. However, the offset of data for the last incident energy will be `N_E(1) - 1` (this is the same as saying the offset for the first incident energy is 0).

The last pdf/cdf value are actually given. I was just looking through lwtr.20t in ENDF71SaB and I do see the pdf=0.0, cdf=1.0 points on each of the distributions. I also confirmed that `L(i+1) = L(i) + N_E(i)*(3 + N_mu)`:
```Python
In [28]: L[:-1] + n_out[:-1]*(3 + n_mu) == L[1:]                                                                                                                                      
Out[28]: 
array([ True,  True,  True,  True,  True,  True,  True,  True,  True,
        True,  True,  True,  True,  True,  True,  True,  True,  True,
        True,  True,  True,  True,  True,  True,  True,  True,  True,
        True,  True,  True,  True,  True,  True,  True,  True,  True,
        True,  True,  True,  True,  True,  True,  True,  True,  True,
        True,  True,  True,  True,  True,  True,  True,  True,  True,
        True,  True,  True,  True,  True,  True,  True,  True,  True,
        True,  True,  True,  True,  True,  True,  True,  True,  True,
        True,  True,  True,  True,  True,  True,  True,  True,  True,
        True,  True,  True,  True,  True,  True,  True,  True,  True,
        True,  True,  True,  True,  True,  True,  True,  True,  True,
        True,  True,  True,  True,  True,  True,  True,  True,  True,
        True,  True,  True,  True,  True,  True,  True])
```